controllers/cluster_controller_test.go

```diff
@@ -52,11 +53,13 @@ func newVsphereClusterReconcilerTest(t *testing.T, objs ...runtime.Object) *vsph
 
 	validator := vsphere.NewValidator(govcClient, &networkutils.DefaultNetClient{}, vcb)
 	defaulter := vsphere.NewDefaulter(govcClient)
+	cniReconciler := vspherereconcilermocks.NewMockCNIReconciler(ctrl)
```

Comment:
Can we create a task to remove/move these tests? If they are still valid, they should be in `reconciler_test.go` :)

Reply:
Yes, will do!